main.go

```diff
@@ -46,5 +50,12 @@ func main() {
 	// Register template functions.
 	beego.AddFuncMap("i18n", i18n.Tr)
 
+	// serve securely if the certificates are present
+	if _, err := os.Open(ServingCertFile); err == nil {
```

Comment:
> Since we don't care about opening the file, could we replace this with os.Stat?

I was pretty sure that `Stat` required that I already have a file: https://godoc.org/os#File.Stat.  Different method I'm missing?